code/javascript/hello.js

```diff
@@ -0,0 +1,24 @@
+// Strings
+const NAME="Mercy"
+console.log(NAME.length)
+console.log(`Hello, my name is  ${NAME}`);
+console.log(`When my friends see me they shout ${NAME.toUpperCase()}!`);
+
+ 
+//Numbers
+
+// Challenge 1
+
+let Number= 15
+Number= Number *2
+Number= Number +8
+Number= Number /2
+Number= Number-15
+console.log(Number);
+
+// Challenge 2
+const radius =7
+const pi=3.14159
+const area = pi * radius * radius
+const roundedArea = Math.round(area*10000)/10000
```

Comment:
I was thinking you'd use the `toFixed()` method for fixed point notation, to get the four digits, but this is a really clever solution!